report/23_Process-CI-CD-Description.md

```diff
@@ -1,2 +1,13 @@
-A complete description of stages and tools included in the CI/CD chains.
-That is, including deployment and release of your systems.
+All CI/CD in the project is achieved using GitHub actions. The different steps of the CI/CD process have been split into workflow files with separate responsibilities:
+- ```dotnetcore.yml``` runs unit tests on the project whenever code is pushed.
+- ```e2e.yml``` runs end-to-end tests when a pull request to master is made.
+- ```sonarPR.yml``` provides a report with code analysis from SonarCloud whenever a pull request to master is made. This report is put on the pull request for easy access.
+- ```sonarMaster.yml``` is triggered when code is pushed to master, and updates the general [SonarCloud report](https://sonarcloud.io/project/issues?id=jlndk_devoops) for the project. This is arguably not part of the CI/CD flow. 
```

Comment:
sonarMaster is a part of the CI flow as it needed for sonarPR to have something to compare to. Without this SonarCloud does not know if the Issues were in master or was added in the new PR that sonarPR scans.